src/caffe/util/gpu_memory.cpp

```diff
@@ -177,15 +148,11 @@ namespace caffe {
       }
 
     CUDA_CHECK(cudaSetDevice(initial_device));
-#if USE_CNMEM
     delete [] devs;
-#endif
```

Comment:
Yes! Sorry my local cuda was a mess at the moment so I only compiled with cpu and left this to Travis. Can you do it or wait another hour ?

Sent using OWA for iPhone
________________________________
From: Luke Yeager <notifications@github.com>
Sent: Friday, November 20, 2015 12:55:05 PM
To: NVIDIA/caffe
Cc: Boris Fomitchev
Subject: Re: [caffe] Removed CNMEM files, removed CMake switch (#81)


In src/caffe/util/gpu_memory.cpp<https://github.com/NVIDIA/caffe/pull/81#discussion_r45517300>:

>      delete [] devs;
> -#endif


Don't you want to delete this line about devs?

-
Reply to this email directly or view it on GitHub<https://github.com/NVIDIA/caffe/pull/81/files#r45517300>.

-----------------------------------------------------------------------------------
This email message is for the sole use of the intended recipient(s) and may contain
confidential information.  Any unauthorized review, use, disclosure or distribution
is prohibited.  If you are not the intended recipient, please contact the sender by
reply email and destroy all copies of the original message.
-----------------------------------------------------------------------------------